README.md

```diff
@@ -56,6 +56,10 @@ If 21 is reached but there is not a lead of 2 points, then the game continues un
 
 A round of defence starts when either the ball is served into play, or the opposing team makes an offensive strike.
 
+Like in standard ping pong, the ball must contact the table before a receiver touches it with their paddle. If the ball hasn't
+yet touched the table, a defender touching the ball before it passes the end of the table will result in a point for the
+offense. Otherwise touching a ball that hasn't bounced will result in a point for the defense.
```

Comment:
This line is confusing. Will touching it before it bounces be a point for the offense or the defense?
Did you mean to say something like
"Otherwise hitting the ball after it has bounced will put it into play, and play will continue as normal"?

Reply:
Agreed - not clear on the last sentence.